Gemfile.lock

```diff
@@ -62,16 +62,16 @@ GEM
     execjs (2.7.0)
     faraday (0.9.2)
       multipart-post (>= 1.2, < 3)
-    faye-websocket (0.10.7)
+    faye-websocket (0.10.9)
       eventmachine (>= 0.12.0)
       websocket-driver (>= 0.5.1)
     ffi (1.9.25)
     gherkin (4.1.3)
     hashdiff (0.3.8)
-    hitimes (1.3.0)
-    i18n (1.5.1)
+    hitimes (1.3.1)
```

Comment:
The build fails because of this, you'd need to manually fix up `Gemfile.lock` to only bump what you need (ZAS and mimemagic)